### ISSUE TITLE
Runtime error: Error(s) in loading state_dict for DetectMultiBackend

### ISSUE THREAD

**Issue by OSMasterSoohwan** [neuralmagic/sparseml#801]
Thank you for solving the last question(The question was that the bounding box was not drawn. I've never deleted a question, but I don't know where it is.)

yolov5s There is no problem with just training it, but there is an error when it is executed by pruned-quantized training through a recipe.

python detect.py --weight yolov5s.pt --source sample.jpeg
python detect.py --weight ./runs/train/origin/weights/best.pt --source sample.jpeg
it's OK, The bounding box is also drawn well.

python detect.py --weight ./runs/train/prunedquantized/weights/best.pt --source sample.jpeg
it occurs
Runtime error: Error(s) in loading state_dict for DetectMultiBackend

If i removed the model.eval() from the export.py file, it doesn't occur error and run, but the bounding box not be drawn.
If i add this eval() again, this error appears. Can you help me?

(I am currently training again at the GPU in case there is something wrong with learning.)

**Comment by KSGulin** [comment]
Thanks for catching this @OSMasterSoohwan! It seems that turning on eval mode before loading the state dict is not good practice, primarily due to issues with loading batch norm weights. Solution is simply to move the model.eval() to after the state dict load and it's up at neuralmagic/yolov5#47

**Comment by OSMasterSoohwan** [comment]
Thank you. I think the problem is solved well.
